app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -79,13 +112,15 @@ class PoliciesFragmentPresenter @Inject constructor(
       supportsConceptCards = false
     )
   }
-
-  override fun onPolicyPageLinkClicked(policyType: PolicyType) {
-    when (policyType) {
-      PolicyType.PRIVACY_POLICY ->
-        (activity as RouteToPoliciesListener).onRouteToPolicies(PolicyPage.PRIVACY_POLICY)
-      PolicyType.TERMS_OF_SERVICE ->
-        (activity as RouteToPoliciesListener).onRouteToPolicies(PolicyPage.TERMS_OF_SERVICE)
-    }
+  private fun updatePolicyDescriptionTvWithParsedHtml(binding: PoliciesFragmentBinding, policyDescription: String){
```

Comment:
Nit: Naming.
```suggestion
  private fun updatePolicyTextViewWithParsedHtml(binding: PoliciesFragmentBinding, policyDescription: String){
```

Reply:
@adhiamboperes  i have made the specified changes, please check.